CopilotKit/.prettierignore

```diff
@@ -2,4 +2,6 @@ README.md
 .next
 .nuxt
 node_modules
-dist
\ No newline at end of file
+dist
+packages/react-core/src/hooks/use-copilot-action.ts
```

Comment:
Great stuff!

Just documenting our discussion comments here for convenience:
- instead of removing both of these files from prettier, let's make the minimal amount of code needed to prettier-ignore, and put it in 2 distinct files
    - `useCopilotAction`: straightforward, just move the hook function
    - `addAction`:  create a base class for the implementation, and extend it with a light subclass that only exposes a different interface (`const ...` constructor)
    - Add bug repot ticket to prettier itself